client/common/wrappers.tsx

```diff
@@ -417,10 +418,24 @@ export const CustomSetHead: React.FC<{ customSet?: CustomSet | null }> = ({
 export const DamageTypeToggle: React.FC<{
   readonly showRanged: boolean;
   readonly setShowRanged: React.Dispatch<React.SetStateAction<boolean>>;
-}> = ({ showRanged, setShowRanged }) => {
+  readonly rangedOnly: boolean;
+  readonly meleeOnly: boolean;
+}> = ({ showRanged, setShowRanged, rangedOnly, meleeOnly }) => {
   const { t } = useTranslation('weapon_spell_effect');
   const theme = useTheme();
 
+  const notPossibleWarning = (
+    <span css={{ marginLeft: 8}}>
```

Comment:
Could you setup prettier if you don't have it or check your configuration? I have it set to format on save, it's amazing

Reply:
Good catch, looks like I didn't have it set up. Formatting on save now, good idea